engine/wscript

```diff
@@ -165,8 +165,10 @@ def build(bld):
 			'client/*.c',
 			'client/vgui/*.c',
 			'client/avi/*.c'])
+	
+		libs.append('opus')
 
-	includes = ['common', 'server', 'client', 'client/vgui', 'tests', '.', '../public', '../common', '../filesystem', '../pm_shared' ]
+	includes = ['common', 'server', 'client', 'client/vgui', 'tests', '.', '../public', '../common', '../filesystem', '../pm_shared', '../3rdparty/opus/opus/include' ]
```

Comment:
Make opus include path is part of it's uselib, so it will be added automatically by waf. See `sdl2.py` for manual usage of uselib.

Reply:
I tried:
```
def configure(conf):
	if conf.check_pkg('opus', 'OPUS', OPUS_CHECK, fatal = False):
		conf.env.HAVE_OPUS = True
		return

	if not conf.path.find_dir('opus') or not conf.path.find_dir('opus/src'):
		conf.fatal('Can\'t find opus submodule. Run `git submodule update --init --recursive`.')
		return

	conf.env.INCLUDES_OPUS = [
		os.path.abspath('opus/include'),
	]
```
But this doesn't work, what I do wrong?

Reply:
Ah, I forgot you're compiling it yourself. Sure, you can do this way.

But you also have export_includes set up already. It should work just right away, I even tested building, and it works without changing include paths for engine.